mojaloop/values.yaml

```diff
@@ -8828,6 +8858,8 @@ ml-ttk-test-val-gp:
     ttkBackendURL: http://$release_name-ml-testing-toolkit-backend:5050
     testSuiteName: GP Tests
     environmentName: Development
+    saveReport: true
+    saveReportBaseUrl: "http://testing-toolkit-backend-ingress-name.local"
```

Comment:
Hi @kleyow , I think we can specify the value as "testing-toolkit.local" in mojaloop values file to match with the default ingress configuration in ml-testing-toolkit. It is just to support local deployments with default values.